tests/validate_config_test.py

```diff
@@ -0,0 +1,254 @@
+# *****************************************************************
```

Comment:
It seems like some of this is duplicated in the build_env tests. Should some of it be moved to a common file?